src/app/components/surveillance/complaints/complaint-edit.jsx

```diff
@@ -455,21 +459,33 @@ function ChplComplaintEdit(props) {
                 error={formik.touched.oncComplaintId && !!formik.errors.oncComplaintId}
                 helperText={formik.touched.oncComplaintId && formik.errors.oncComplaintId}
               />
-              <Select
-                multiple
-                id="complaint-types"
-                name="complaintTypes"
-                label="Complaint Types"
-                required
-                value={formik.values.complaintTypes}
-                onChange={formik.handleChange}
-                onBlur={formik.handleBlur}
-                error={formik.touched.complaintTypes && !!formik.errors.complaintTypes}
-              >
-                { complaintTypes.map((item) => (
-                  <MenuItem value={item} key={item.id}>{item.name}</MenuItem>
-                ))}
-              </Select>
+              <FormControl variant="outlined">
+                <InputLabel id="complaint-types">Compliant Types</InputLabel>
+                <Select
+                  multiple
+                  id="complaint-types"
```

Comment:
Are these supposed to have the same `id` value? If they are, great, but I thought `id`s were supposed to be unique on a page

Reply:
https://v4.mui.com/components/selects/ Using htmlFor after reviewing MUI docs